libraries/stdlib/jvm/test/random/RandomSerializationTest.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package random
+
+import test.io.deserializeFromHex
+import test.io.serializeAndDeserialize
+import kotlin.random.Random
+import kotlin.random.asJavaRandom
+import kotlin.random.asKotlinRandom
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertSame
+
+class RandomSerializationTest {
+    @Test
+    fun defaultIsSerializable() {
+        val instance = Random
+        assertSame(instance, serializeAndDeserialize(instance))
```

Comment:
@ilya-g this test already checks that Random.default instances are same when serialized.

Reply:
It doesn't catch the situation when we change the default instance in an incompatible way.

Reply:
In the current state of changes, `default` has no serialized fields and has constant `serialVersionUID`. Instances will be persistent until we add non-transient field or change `serialVersinoUID`. I'm not sure how to test such a situation well.